sigmf-spec.md

```diff
@@ -228,7 +228,7 @@ the `global` object:
 |`description`|false|string|A text description of the SigMF recording.|
 |`author`|false |string|The author's name (and optionally e-mail address).|
 |`date`|false|string|An ISO-8601 formatted string of the capture date of the recording.|
-|`license`|false|string|The license under which the recording is offered.|
+|`license`|false|string|A URL for the license document under which the recording is offered; when possible, use the canonical document provided by the license author.|
```

Comment:
“provided by the license author or, failing that, a well-known one.”

For example, the traditional open source licenses were first used on individual projects and then people reused them without that being the intent of the author.